src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Ball {
+    String color;
+    int number;
```

Comment:
Class variables must be private
```suggestion
    private String color;
    private int number;
```